js/src/forum/components/Post.js

```diff
@@ -133,11 +133,11 @@ export default class Post extends Component {
       classes.push('Post--loading');
     }
 
-    if (user && user === app.session.user) {
+    if (user === app.session.user) {
```

Comment:
Doesn't this prevent the class being applied on posts with deleted users while browsing as guest?

Reply:
Wow, I would have never thought of that...
```suggestion
    if (user && user === app.session.user) {
```